interp/constrextern.ml

```diff
@@ -551,7 +551,7 @@ and extern_notation_pattern allscopes vars t = function
   | [] -> raise No_match
   | (keyrule,pat,n as _rule)::rules ->
     try
-      if is_inactive_rule keyrule then raise No_match;
+      if is_inactive_rule keyrule || is_printing_inactive_rule keyrule pat then raise No_match;
```

Comment:
Maybe we should include `is_inactive_rule` in `is_printing_inactive_rule`?

Reply:
This is temporary: `is_inactive_rule` will disappear in #12324.